system/default_ad_api/src/interface.cpp

```diff
@@ -12,27 +12,26 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "default_ad_api/nodes/interface_version.hpp"
+#include "interface.hpp"
```

Comment:
I believe you should split this refactoring into another PR for reviewers.

Reply:
Yes. That is preferable. However, this refactoring depends on this PR. It can't be separated because the build fails.

Reply:
Okay, I understand!